test/distributed/fsdp/test_fsdp_grad_acc.py

```diff
@@ -132,8 +162,13 @@ def permute_tensor(x: torch.Tensor):
 
     @skip_if_lt_x_gpu(2)
     @parametrize(
-        "num_iters_to_acc",
-        [2, 4],
+        "configs",
+        [
+            [_GradAccConfig(True, 4)],
```

Comment:
nit: if you want, maybe pass in named args here, so future developers know what it is